.github/workflows/cmake.yml

```diff
@@ -16,7 +16,7 @@ jobs:
           compiler_prefix: /usr/bin
         - compiler_driver: g++-10
           compiler_prefix: /usr/bin
-        - compiler_driver: clang++-12
+        - compiler_driver: clang++
```

Comment:
What version does the latest ubuntu comes with?

Reply:
From the logs
```
-- The CXX compiler identification is Clang 14.0.0

Reply:
In my initial push I had changed it to `clang++-14` but that had failed too.